vdev/src/git.rs

```diff
@@ -51,6 +75,21 @@ pub fn list_files() -> Result<Vec<String>> {
         .collect())
 }
 
+fn branch_exists(branch_name: &str) -> Result<bool> {
+    let output = capture_output(&["rev-parse", "--verify", branch_name])?;
+    Ok(!output.is_empty())
+}
+
+fn checkout_branch_to_existing(branch_name: &str) -> Result<()> {
```

Comment:
… and then this would be the plain `checkout branch`.

Reply:
I have implemented these changes